model/src/w3iogoncdmd.F90

```diff
@@ -34,9 +34,9 @@ subroutine W3IOGONCD ()
     USE W3ADATMD, ONLY: CFLXYMAX, CFLTHMAX, CFLKMAX, P2SMS, US3D
     USE W3ADATMD, ONLY: TH1M, STH1M, TH2M, STH2M, HSIG, PHICE, TAUICE
     USE W3ADATMD, ONLY: STMAXE, STMAXD, HMAXE, HCMAXE, HMAXD, HCMAXD, USSP
-#ifdef W3_CESMCOUPLED
-    USE W3ADATMD, ONLY: LANGMT
-#endif
+ #ifdef W3_CESMCOUPLED
```

Comment:
Please remove the whitespaces before ifdefs.

Reply:
Done